<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImfaceWindow</class>
 <widget class="QWidget" name="ImfaceWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>960</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>960</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>960</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ModelFitting</string>
  </property>
  <widget class="QGroupBox" name="view_region">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>960</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>960</width>
     <height>960</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>960</width>
     <height>960</height>
    </size>
   </property>
   <property name="title">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="CModelViewer" name="model_viewer" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>960</width>
        <height>960</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>960</width>
        <height>960</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QSlider" name="base_cut_slilder">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>50</y>
     <width>22</width>
     <height>241</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-50</number>
   </property>
   <property name="maximum">
    <number>49</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QSlider" name="region_threshold_slilder">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>50</y>
     <width>22</width>
     <height>241</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>99</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="pageStep">
    <number>10</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="id_2">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>500</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="id_3">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>500</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="id_4">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>500</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="id_5">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>500</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>570</y>
     <width>221</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>GroupBox</string>
   </property>
   <widget class="QPushButton" name="LoadImage">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>load image</string>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="id_6">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>500</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="id_1">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>500</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_template_kind">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>530</y>
     <width>201</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="id_7">
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>500</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CModelViewer</class>
   <extends>QWidget</extends>
   <header>cmodelviewer.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
